{{cookiecutter.project_slug}}/project/settings/base.py

```diff
@@ -312,13 +312,19 @@
 BASE_URL = os.environ.get("WAGTAIL_BASE_URL", "")
 WAGTAIL_ENABLE_UPDATE_CHECK = False
 WAGTAIL_REDIRECTS_FILE_STORAGE = "cache"
-WAGTAILSEARCH_BACKENDS = {
-    "default": {
-        "BACKEND": "wagtail.search.backends.elasticsearch6",
-        "URLS": [os.environ.get("SEARCH_URL", "http://127.0.0.1:9200")],
-        "INDEX": os.environ.get("SEARCH_INDEX", f"{PROJECT_SLUG}"),
+
+# Wagtail search - use Postgres unless ENABLE_ELASTICSEARCH is enabled with environment vars
```

Comment:
Could you remove the toggle?

I think given the... fun.... we've had with elasticsearch - I think removal by default is the better route. So just remove this as a section, and move WAGTAILSEARCH_BACKENDS up.